content/uiux/concepts/design-component/design-component.md

```diff
@@ -0,0 +1,17 @@
+---
+Title: 'Design Component'
+Description: 'Design components are specific examples of design elements that might exist within a given context.'
```

Comment:
This is for SEO and is the little blurb that shows up in google under the link. It'd probably be good to fit in mention of design systems and a few examples (buttons, icons, form fields, etc.)